static/code/objectPanel.js

```diff
@@ -126,6 +126,7 @@ Code.ObjectPanel.TYPES = {
   'array': '[]',
   'boolean': '⏼',
   'function': '𝑓',
+  'null': '␀',
```

Comment:
Oh, nice.  I was going to suggest using "∅", but—provided it is _legible_—this is even more clear.

Reply:
I too have concerns on that front.  Chrome renders this character tiny with lots of surrounding whitespace.  Firefox is better.  And Android renders 'NUL' diagonally.  So much diversity!

Whatever, we can always change it.

Reply:
> Whatever, we can always change it.

My thought exactly.